### PR TITLE
Default to opening all Scratch file formats

### PR DESCRIPTION
**Pull request by doyousketch2**
Defaults to opening all Scratch file formats.

Viewable online here - http://s000.tinyupload.com/index.php?file_id=21922643149425089004

File > Load Project - to view changes.

### CODE REVIEW

File: src/scratch/ScratchRuntime.as [hunk 1]
```diff
@@ -370,11 +370,12 @@ public class ScratchRuntime {
 		stopAll();
 		var fileList:FileReferenceList = new FileReferenceList();
 		fileList.addEventListener(Event.SELECT, fileSelected);
-		var filter1:FileFilter = new FileFilter('Scratch 1.4 Project', '*.sb');
-		var filter2:FileFilter = new FileFilter('Scratch 2 Project', '*.sb2');
+		var filter0:FileFilter = new FileFilter('All Scratch Projects', '*.sb?');
```
Comment (nathan) [inline src/scratch/ScratchRuntime.as]: `"*.sb;*.sb2"` would probably be better here.

**Comment by AndersBillLinden** [comment]
Is the pull request doing the same as the accepted https://github.com/LLK/scratch-flash/pull/618 ?

**Comment by sclements** [comment]
Already addressed by #618 but if we need multiple file filters in the future we'll have to revert the signature change.

**Comment by sclements** [comment]
BTW, Github is having issues and was reopening this PR right after closing it. I'm not crazy! :-1:

**Pull request closed without merging**